### DISCUSSION TITLE
order of events is serialized randomly?

### DISCUSSION THREAD

**Discussion by jablka** [ics-py/ics-py#397] [Q&A] [answered]
hello,
I have noticed in final `.ics` file that the order by which events are serialized is completely random? 
Why so?
(Or maybe I understand: those Event objects are like a hash-table where order is not guaranteed... ?)

**Comment by N-Coder** [comment, marked as answer]
In version 0.7 we used a set, in version 0.8 (which is currently in alpha), this was changed to a list with a user-defined ordering.
